### PR TITLE
v2.2.0 – Separating out Fozzie specific documentation into general docs (from CSS section)

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- Added new page to 'General' section explaining what Fozzie is.  Have moved some of this content out of the CSS section as it was relevant to all aspects of the components.

### Changed
- Package updates for `fozzie`, `f-header`, `f-footer`, `f-icons`

### CODE REVIEW

File: docs/src/templates/pages/styleguide/ui-components/alerts.hbs [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
```
Comment (DamianMullins) [inline docs/src/templates/pages/styleguide/ui-components/alerts.hbs]: This looks like an exact copy of [the badges page](https://github.com/justeat/global-component-library/blob/master/docs/src/templates/pages/styleguide/ui-components/badges.hbs)?

Comment (ashleynolan) [reply docs/src/templates/pages/styleguide/ui-components/alerts.hbs]: Yep – I wasn't meant to commit this… 🤦‍♂️

Comment (ashleynolan) [reply docs/src/templates/pages/styleguide/ui-components/alerts.hbs]: Have removed this now – good catch ⚾️

File: README.md [hunk 2]
```diff
@@ -9,3 +9,19 @@
 ---
 
 [![Build Status](https://travis-ci.org/justeat/global-component-library.svg)](https://travis-ci.org/justeat/global-component-library)
+
+## Usage
+
+To get up and running with the Fozzie Component Library, follow these steps:
+
+1. Clone this repo to your local machine `git clone https://github.com/justeat/global-component-library.git`
+2. In your command prompt, run `yarn` from the base of the project. This will install all of the necessary project dependencies.
```
Comment (kevinrodrigues) [inline README.md]: Should we include a step about installing yarn. Same for the next one, step for gulp?

Comment (ashleynolan) [reply README.md]: I'm working up another docs PR now – so will add this in

**Pull request merged**